dnf-behave-tests/dnf/steps/history.py

```diff
@@ -73,10 +73,27 @@ def history_equal(history, table):
 
 
 @behave.then('stdout is history list')
-def step_impl(context):
+def then_stdout_is_history_list(context):
     assert_history_list(context, context.cmd_stdout)
 
 
+@behave.then("{dnf_version:dnf_version} stdout is history list")
```

Comment:
Is `{dnf_version:dnf_version}` really necessary? Isn't `{dnf_version}` enough?

Reply:
I thing it is good. What's confusing here is that the second `dnf_version` is a type with parser registered here: https://github.com/rpm-software-management/ci-dnf-stack/blob/5295644b1b6652765aefbe58341a8f6dba9ea2d3/dnf-behave-tests/common/output.py#L110
Having this type enforces that only `dnf4` and `dnf5` values of this parameter are accepted.